doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -30,8 +30,8 @@ It implements the nRF Wi-Fi driver software on the nRF5340 host processor commun
 The nRF5340 host is a dual-core SoC based on the Arm® Cortex®-M33 architecture.
 It has the following features:
 
-* A full-featured Arm Cortex-M33F core with DSP instructions, FPU, and Armv8-M Security Extension, running at up to 128 MHz, referred to as the *application core*.
-* A secondary Arm Cortex-M33 core, with a reduced feature set, running at a fixed 64 MHz, referred to as the *network core*.
+* A full-featured Arm Cortex-M33F core with DSP instructions, FPU, and Armv8-M Security Extension, running at up to 128 MHz, referred to as the application core.
```

Comment:
I believe there should be trademarks, as there is: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf70/gs.html#overview
```suggestion
* A full-featured Arm® Cortex®-M33F core with DSP instructions, FPU, and Armv8-M Security Extension, running at up to 128 MHz, referred to as the application core.
```

Reply:
I think it's on line 30, at the first instance.